src/sorting.jl

```diff
@@ -102,13 +102,13 @@ Partition the region of `values` after index `lo` up to (inclusive) `hi` with
 respect to `pivot`. This is done by a 'binary' merge sort, where each the values
 are sorted by a boolean key: how they compare to `pivot`. The comparison is
 affected by `parity`. See `flex_lt`. `swap` is an array for exchanging values
-and `sums` is an array of Int32s used during the merge sort.
```

Comment:
@xaellison did you have a specific reason for using Int32? I didn't notice performance degradation by switching to Int (occupancy was identical despite the increased shared memory usage), and it avoids exceptions paths due to Int<->Int32 conversions.

Reply:
That was just for reducing shmem usage. If the overall performance isn't worse, Int makes sense instead of Int32.